### PR TITLE
Allow motions attorney to recommend partial grant

### PR DESCRIPTION
**Pull request by nanotone**
Connects #13864 

### Description
This PR adds an option for motions attorneys to recommend partially granting vacatur to the judge, without selecting which issues to recommend granting.

### Acceptance Criteria
- [x] Please put this work behind the feature toggle: [review_motion_to_vacate]
- [x] Motions attorneys have a radio field for "Grant partial vacatur" when selecting a recommendation to send to the judge
  - [x] Show same help text as judge address motion to vacate task
  - [x] Do not show issues for selection, just the radio button
- [x] The partial grant selection is indicated in the instructions to the judge
- [x] Remove "required" labels from fields in the Motion to Vacate flow (not related to this specific request, but a small adder from dogfooding)
- [x] Double check that code doesn't have the word "vacature" (with an e) anywhere
- [x] Include screenshot(s) in the Github issue
- [ ] Update documentation: [Motions to Vacate](https://github.com/department-of-veterans-affairs/caseflow/wiki/Motions-to-Vacate)

### Testing Plan
1. Set up a Vacate motion mail task for a motions attorney to handle
2. As the motions attorney, start the Send to Judge flow for this task
3. Select the partial grant radio button
4. New feature spec added

### User Facing Changes
Screen Shot 2020-04-09 at 15 27 24

### CODE REVIEW

File: spec/feature/queue/motion_to_vacate_spec.rb [hunk 1]
```diff
@@ -843,7 +854,7 @@ def verify_cavc_conflict_action
     action_dropdown.click
   end
 
-  def select_issue_for_vacature(issue_id)
+  def select_issue_for_vacatur(issue_id)
```
Comment (leikkisa) [inline spec/feature/queue/motion_to_vacate_spec.rb:857]: Nice find! 😺

File: client/app/queue/mtv/MTVDispositionSelection.jsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { DISPOSITION_OPTIONS, PARTIAL_GRANT_OPTION } from '../../../constants/MO
 const [grant, ...rest] = DISPOSITION_OPTIONS;
 const optsWithPartial = [grant, PARTIAL_GRANT_OPTION, ...rest];
 
-export const MTVDispositionSelection = ({ label = '', value: initialVal = null, onChange, allowPartial = false }) => {
+export const MTVDispositionSelection = ({ label = '', value: initialVal = null, onChange, allowPartial = true }) => {
```
Comment (leikkisa) [inline client/app/queue/mtv/MTVDispositionSelection.jsx:11]: I think we should remove the allowPartial option altogether, and simplify the code - there's a comment on this file a few lines up saying that we only allow partially granted for judges to explain where there are two versions of the disposition options which is no longer relevant.

The PARTIAL_GRANT_OPTION can join its friends in the DISPOSITION_OPTIONS constant now.

File: spec/feature/queue/motion_to_vacate_spec.rb [hunk 3]
```diff
@@ -114,17 +114,21 @@
...
+        fill_in("instructions", with: "Attorney context/instructions for judge")
 
-        # Verify new task was created
-        judge_task = JudgeAddressMotionToVacateTask.find_by(assigned_to: judge2)
-        expect(judge_task).to_not be_nil
+        # Ensure it has pre-selected judge previously assigned to case
+        expect(dropdown_selected_value(find(".dropdown-judge"))).to eq judge2.display_name
+
+        task = submit_and_fetch_task(judge2)
+        expect(task.instructions.first).to include("I recommend granting a partial vacatur")
```
Comment (leikkisa) [inline spec/feature/queue/motion_to_vacate_spec.rb:131]: 👍

File: client/app/queue/mtv/MTVDispositionSelection.jsx (hunk 2 above)
Comment (nanotone) [reply client/app/queue/mtv/MTVDispositionSelection.jsx:11]: >can join its friends
😭 too real, Sally.

File: client/constants/MOTION_TO_VACATE.json [hunk 4]
```diff
@@ -18,6 +18,11 @@
       "displayText": "Grant all issues for vacatur",
       "value": "granted"
     },
+    {
```
Comment (jcq) [inline client/constants/MOTION_TO_VACATE.json:21]: I like that this PR allows us to remove the kludgy way this was structured before!

File: client/app/queue/mtv/MotionsAttorneyDisposition.jsx [hunk 5]
```diff
@@ -59,6 +59,11 @@ export const MotionsAttorneyDisposition = ({ judges, selectedJudge, task, appeal
     return true;
   };
 
+  const instructionsLabel = sprintf(
```
Comment (jcq) [inline client/app/queue/mtv/MotionsAttorneyDisposition.jsx:62]: 👍 on moving this logic out of the render

File: spec/feature/queue/motion_to_vacate_spec.rb (hunk 1 above)
Comment (jcq) [reply spec/feature/queue/motion_to_vacate_spec.rb:857]: Haha, I bet this was written before we realized the correct spelling (or at least before my brain fully internalized it)!

File: spec/feature/queue/motion_to_vacate_spec.rb [hunk 6]
```diff
@@ -114,17 +114,21 @@
         # Ensure it has pre-selected judge previously assigned to case
         expect(dropdown_selected_value(find(".dropdown-judge"))).to eq judge2.display_name
 
-        click_button(text: "Submit")
+        task = submit_and_fetch_task(judge2)
+        expect(task.instructions.first).to include("I recommend granting a vacatur")
```
Comment (jcq) [inline spec/feature/queue/motion_to_vacate_spec.rb:118]: Oh, this is easier than building out the entire set of formatted instructions and checking the full thing for equivalence. I... huh. Wish I had done that for just the things that are different. :)

File: client/app/queue/mtv/MTVJudgeDisposition.jsx [hunk 7]
```diff
@@ -130,7 +130,6 @@ export const MTVJudgeDisposition = ({
 
         <MTVDispositionSelection
           label={JUDGE_ADDRESS_MTV_DISPOSITION_SELECT_LABEL}
-          allowPartial
```
Comment (jcq) [inline client/app/queue/mtv/MTVJudgeDisposition.jsx:133]: Again, yay for removing kludge!

**Pull request merged**